src/app/space-wizard/components/wizard-dynamic-step/wizard-dynamic-step.component.ts

```diff
@@ -1,66 +1,72 @@
-import { Component, OnInit, OnChanges, OnDestroy, SimpleChanges, SimpleChange, Input, Output, Inject, EventEmitter } from '@angular/core';
-import { Router } from '@angular/router';
+import { Component, Inject, Input, OnChanges, OnDestroy, OnInit, SimpleChanges } from '@angular/core';
 
-import { IWorkflowStep, IWorkflowTransition, IWorkflow, WorkflowTransitionDirection } from '../../models/workflow';
-import { LoggerFactory, ILoggerDelegate } from '../../common/logger';
-import { INotifyPropertyChanged } from '../../core/component'
+import { IMultiSelectSettings, IMultiSelectTexts } from 'angular-2-dropdown-multiselect';
+import { ILoggerDelegate, LoggerFactory } from '../../common/logger';
+import { INotifyPropertyChanged } from '../../core/component';
 
-import { IFieldInfo, IFieldSet, IAppGeneratorResponse, IAppGeneratorRequest, IAppGeneratorCommand, IAppGeneratorService, IAppGeneratorServiceProvider } from '../../services/app-generator.service';
+import { IWorkflow, IWorkflowTransition, WorkflowTransitionDirection } from '../../models/workflow';
 
-
-import { IMultiSelectOption, IMultiSelectSettings, IMultiSelectTexts } from 'angular-2-dropdown-multiselect';
+import {
+  IAppGeneratorRequest,
+  IAppGeneratorResponse,
+  IAppGeneratorService,
+  IAppGeneratorServiceProvider,
+  IFieldSet
+} from '../../services/app-generator.service';
 
 @Component({
-  host: {
-    'class': 'wizard-step'
-  },
-  selector: 'wizard-dynamic-step',
-  templateUrl: './wizard-dynamic-step.component.html',
-  styleUrls: ['./wizard-dynamic-step.component.scss'],
-  providers: [IAppGeneratorServiceProvider.FactoryProvider]
-})
+             host       : {
```

Comment:
this indent doesn't look right, if it is in the linter then we need to update the linter.